tests/Microsoft.Identity.Test.Unit/PublicApiTests/PublicClientApplicationTests.cs

```diff
@@ -1263,6 +1266,35 @@ public void CheckUserProvidedCorrelationIDTest()
             }
         }
 
+#if NET_CORE
+        [TestMethod]
+        public void CheckRuntimeBrokerSupportTest()
```

Comment:
nit: rename to clarify that this tests the pop method, and move to pop tests class?